src/Test/Utilities/Portable/TestUtilities.csproj

```diff
@@ -46,20 +45,16 @@
     <PlatformTarget>ARM</PlatformTarget>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|Any CPU' ">
-    <PlatformTarget>
-    </PlatformTarget>
+    <PlatformTarget></PlatformTarget>
```

Comment:
Not directly related to your change, but this might be something else you'll want to flag. This seems hilariously dangerous.